chirp/drivers/ft4.py

```diff
@@ -308,16 +308,26 @@ def startcomms(radio, way):
     progressbar.max = radio.numblocks
     enter_clonemode(radio)
     id_response = sendcmd(radio.pipe, b'\x02', None)
+
+    # The last byte of the ID contains info about regional differences
+    radio.subtype = id_response[len(id_response) - 1]
```

Comment:
Hmm, this will work for the case where you download the image, make changes and upload, but it won't work if I save the image and then re-open it right? Won't we be using the default to calculate the multiplier in that case?

I'm guessing that we need to have another radio class for the asia variant, and abort the clone here if the class they chose doesn't match the radio's ID string. Then we'll always know which multiplier to use based on the class, and we'll make sure we're talking to the right radio whenever we do.